--- tuxpaint-0.9.17/Makefile~	2007-07-03 08:44:39.000000000 +0200
+++ tuxpaint-0.9.17/Makefile	2007-07-03 08:49:33.000000000 +0200
@@ -441,14 +441,14 @@
 
 uninstall:	uninstall-i18n
 	-if [ "x$(GNOME_PREFIX)" != "x" ]; then \
-	  rm $(GNOME_PREFIX)/share/gnome/apps/Graphics/tuxpaint.desktop; \
+	  rm $(GNOME_PREFIX)/share/gnome/apps/tuxpaint.desktop; \
 	  rm $(GNOME_PREFIX)/share/pixmaps/tuxpaint.png; \
 	else \
-	  rm /usr/share/gnome/apps/Graphics/tuxpaint.desktop; \
+	  rm /usr/share/gnome/apps/tuxpaint.desktop; \
 	  rm /usr/share/pixmaps/tuxpaint.png; \
 	fi
 	-if [ "x$(KDE_PREFIX)" != "x" ]; then \
-	  rm $(KDE_PREFIX)/Graphics/tuxpaint.desktop; \
+	  rm $(KDE_PREFIX)/tuxpaint.desktop; \
 	fi
 	-rm $(ICON_PREFIX)/tuxpaint.png
 	-rm $(X11_ICON_PREFIX)/tuxpaint.xpm
@@ -512,9 +512,9 @@
 	 install -d $(PKG_ROOT)$(GNOME_PREFIX)/share/pixmaps; \
 	 cp data/images/icon.png $(PKG_ROOT)/$(GNOME_PREFIX)/share/pixmaps/tuxpaint.png; \
 	 chmod 644 $(PKG_ROOT)$(GNOME_PREFIX)/share/pixmaps/tuxpaint.png; \
-	 install -d $(PKG_ROOT)$(GNOME_PREFIX)/share/gnome/apps/Graphics; \
-	 cp src/tuxpaint.desktop $(PKG_ROOT)$(GNOME_PREFIX)/share/gnome/apps/Graphics/; \
-	 chmod 644 $(PKG_ROOT)$(GNOME_PREFIX)/share/gnome/apps/Graphics/tuxpaint.desktop; \
+	 install -d $(PKG_ROOT)$(GNOME_PREFIX)/share/gnome/apps/; \
+	 cp src/tuxpaint.desktop $(PKG_ROOT)$(GNOME_PREFIX)/share/gnome/apps/; \
+	 chmod 644 $(PKG_ROOT)$(GNOME_PREFIX)/share/gnome/apps/tuxpaint.desktop; \
 	fi
 
 
@@ -549,9 +549,9 @@
 	@echo
 	@echo "...Installing launcher icon into KDE..."
 	@if [ "x$(KDE_PREFIX)" != "x" ]; then \
-	  install -d $(PKG_ROOT)$(KDE_PREFIX)/Graphics; \
-	  cp src/tuxpaint.desktop $(PKG_ROOT)$(KDE_PREFIX)/Graphics/; \
-	  chmod 644 $(PKG_ROOT)$(KDE_PREFIX)/Graphics/tuxpaint.desktop; \
+	  install -d $(KDE_PREFIX); \
+	  cp src/tuxpaint.desktop $(KDE_PREFIX); \
+	  chmod 644 $(KDE_PREFIX)/tuxpaint.desktop; \
 	fi
 
 
--- tuxpaint-0.9.17/src/tuxpaint.desktop~	2007-05-09 09:46:21.000000000 +0200
+++ tuxpaint-0.9.17/src/tuxpaint.desktop	2007-07-03 08:52:39.000000000 +0200
@@ -182,3 +182,7 @@
 Comment[xh]=Inkqubo yokuzoba yabantwana.
 Comment[zh_cn]=孩子的绘图程序。
 Comment[zh_tw]=適合兒童的畫圖程式
+Categories=Education;Art;
+# vi: set encoding=utf-8
+
+
