--- tuxpaint-0.9.20/src/tuxpaint.desktop	2008-06-27 20:15:41.000000000 +0000
+++ tuxpaint-0.9.20/src/tuxpaint.desktop	2008-09-03 15:15:22.000000000 +0000
@@ -49,7 +49,7 @@
 Name[oc]=Tux Paint
 Name[pl]=Tux Paint
 Name[pt_BR]=Tux Paint
-Name[pt_PT]=Tux Paint
+Name[pt]=Tux Paint
 Name[ru]=Рисуй вместе с Tux!
 Name[sk]=Kreslenie s Tuxom
 Name[sl]=Tux Paint
@@ -125,7 +125,7 @@
 GenericName[oc]=Logicial de dessenh
 GenericName[pl]=Program do rysowania
 GenericName[pt_BR]=Programa de desenho
-GenericName[pt_PT]=Programa de desenho
+GenericName[pt]=Programa de desenho
 GenericName[ru]=Программа для рисования
 GenericName[sk]=Kresliaci program
 GenericName[sl]=Risarski program
@@ -195,7 +195,7 @@
 Comment[nr]=Iphrogremu yokudweba yabantwana.
 Comment[pl]=Program do rysowania dla dzieci.
 Comment[pt_BR]=Um programa de desenho para crianças.
-Comment[pt_PT]=Um programa de desenho para crianças.
+Comment[pt]=Um programa de desenho para crianças.
 Comment[ru]=Детская программа для рисования.
 Comment[sk]=Kresliaci program pre deti.
 Comment[sl]=Risarski program za otroke
@@ -216,3 +216,4 @@
 Comment[zh_CN]=孩子的绘图程序。
 Comment[zh_TW]=適合兒童的畫圖程式
 Comment[zw]=Diif program paar keé men Dibuj paar biét biss
+Categories=Education;Art;
