--- tuxpaint-0.9.16/Makefile~	2006-10-23 21:15:11.000000000 +0200
+++ tuxpaint-0.9.16/Makefile	2006-10-23 21:17:55.000000000 +0200
@@ -876,10 +876,10 @@
 	@cp trans/pl.mo $(LOCALE_PREFIX)/pl/LC_MESSAGES/tuxpaint.mo
 	@chmod 644 $(LOCALE_PREFIX)/pl/LC_MESSAGES/tuxpaint.mo
 	@#
-	@echo "   pt_PT ...Portuguese (Portugal)..."
-	@install -d $(LOCALE_PREFIX)/pt_PT/LC_MESSAGES
-	@cp trans/pt_pt.mo $(LOCALE_PREFIX)/pt_PT/LC_MESSAGES/tuxpaint.mo
-	@chmod 644 $(LOCALE_PREFIX)/pt_PT/LC_MESSAGES/tuxpaint.mo
+	@echo "   pt ...Portuguese (Portugal)..."
+	@install -d $(LOCALE_PREFIX)/pt/LC_MESSAGES
+	@cp trans/pt_pt.mo $(LOCALE_PREFIX)/pt/LC_MESSAGES/tuxpaint.mo
+	@chmod 644 $(LOCALE_PREFIX)/pt/LC_MESSAGES/tuxpaint.mo
 	@#
 	@echo "   ro_RO ...Romanian..."
 	@install -d $(LOCALE_PREFIX)/ro/LC_MESSAGES
--- tuxpaint-0.9.16/src/tuxpaint.desktop~	2006-10-23 21:15:11.000000000 +0200
+++ tuxpaint-0.9.16/src/tuxpaint.desktop	2006-10-23 21:22:42.000000000 +0200
@@ -11,9 +11,9 @@
 Name[cy]=Tux Paint
 Name[da]=Tux maling
 Name[de]=Tux Paint
-Name[en_gb]=Tux Paint
+Name[en_GB]=Tux Paint
 Name[es]=Tux Paint
-Name[es_mx]=Tux Paint
+Name[es_MX]=Tux Paint
 Name[et]=Joonistame koos Tuksiga!
 Name[eu]=Tux Paint
 Name[fi]=Tux Paint
@@ -36,8 +36,8 @@
 Name[nl]=TuxPaint
 Name[nn]=Tux Paint
 Name[pl]=Tux Paint
-Name[pt_br]=Pintar
-Name[pt_pt]=Tux Paint
+Name[pt_BR]=Pintar
+Name[pt]=Tux Paint
 Name[ru]=Рисуй вместе с Tux!
 Name[sk]=Tux Paint
 Name[sl]=Tux Paint
@@ -50,8 +50,8 @@
 Name[uk]=Малюй разом з Tux!
 Name[vi]=Tux Sơn
 Name[wa]=Tux Paint
-Name[zh_cn]=Tux Paint
-Name[zh_tw]=企鵝小畫家
+Name[zh_CN]=Tux Paint
+Name[zh_TW]=企鵝小畫家
 Type=Application
 Exec=tuxpaint
 Icon=tuxpaint
@@ -67,9 +67,9 @@
 GenericName[cy]=Rhaglen lunio
 GenericName[da]=Tegne program
 GenericName[de]=Malprogramm
-GenericName[en_gb]=Drawing program
+GenericName[en_GB]=Drawing program
 GenericName[es]=Programa de dibujo
-GenericName[es_mx]=Programa de dibujo
+GenericName[es_MX]=Programa de dibujo
 GenericName[et]=Joonistusprogramm
 GenericName[eu]=Marrazketa programa
 GenericName[fi]=Piirto-ohjelma
@@ -93,8 +93,8 @@
 GenericName[nl]=Tekenprogramma
 GenericName[nn]=Teikneprogram
 GenericName[pl]=Program do rysowania
-GenericName[pt_br]=Programa de desenho
-GenericName[pt_pt]=Programa de desenho
+GenericName[pt_BR]=Programa de desenho
+GenericName[pt]=Programa de desenho
 GenericName[ru]=Программа для рисования
 GenericName[sk]=Kresliaci program
 GenericName[sl]=Risarski program
@@ -107,8 +107,8 @@
 GenericName[uk]=Програма для малювання
 GenericName[vi]=Trình vẽ
 GenericName[wa]=Programe di dessinaedje
-GenericName[zh_cn]=绘图程序
-GenericName[zh_tw]=畫圖程式
+GenericName[zh_CN]=绘图程序
+GenericName[zh_TW]=畫圖程式
 Comment=A drawing program for children.
 Comment[af]='n Tekenprogram vir kinders.
 Comment[ar]=.برنامج رسومات للأطفال
@@ -120,9 +120,9 @@
 Comment[cy]=Rhaglen lunio ar gyfer plant.
 Comment[da]=Et tegneprogram for børn.
 Comment[de]=Ein Malprogramm für Kinder.
-Comment[en_gb]=A drawing program for children.
+Comment[en_GB]=A drawing program for children.
 Comment[es]=Un programa de dibujo para niños.
-Comment[es_mx]=Un programa de dibujo para niños.
+Comment[es_MX]=Un programa de dibujo para niños.
 Comment[et]=Joonistusprogramm lastele.
 Comment[eu]=Umeentzako marrazketa programa
 Comment[fi]=Piirto-ohjelma pienille lapsille.
@@ -145,8 +145,8 @@
 Comment[nb]=Et tegneprogram for de yngste.
 Comment[nl]=Een tekenprogramma voor kinderen.
 Comment[nn]=Eit teikneprogram for dei yngste.
-Comment[pt_br]=Um programa de desenho para crianças
-Comment[pt_pt]=Um programa de desenho para crianças.
+Comment[pt_BR]=Um programa de desenho para crianças
+Comment[pt]=Um programa de desenho para crianças.
 Comment[ru]=Детская программа для рисования.
 Comment[sk]=Kresliaci program pre deti.
 Comment[sl]=Risarski program za otroke
@@ -158,7 +158,7 @@
 Comment[tr]=Çocuklar için bir çizim programı
 Comment[vi]=Một chương trình vẽ cho đứa bé.
 Comment[wa]=Programe di dessinaedje po ls efants
-Comment[zh_cn]=孩子的绘图程序。
-Comment[zh_tw]=適合兒童的畫圖程式
+Comment[zh_CN]=孩子的绘图程序。
+Comment[zh_TW]=適合兒童的畫圖程式
 Categories=Education;Art;
 # vi: set encoding=utf-8
