--- tuxpaint-0.9.14/Makefile~	Wed Nov 10 12:54:57 2004
+++ tuxpaint-0.9.14/Makefile	Wed Nov 10 12:54:57 2004
@@ -594,9 +594,9 @@
 	@chmod 644 $(LOCALE_PREFIX)pl/LC_MESSAGES/tuxpaint.mo
 	@#
 	@echo "   pt_PT ...Portuguese (Portugal)..."
-	@install -d $(LOCALE_PREFIX)pt_PT/LC_MESSAGES
-	@cp trans/pt_pt.mo $(LOCALE_PREFIX)pt_PT/LC_MESSAGES/tuxpaint.mo
-	@chmod 644 $(LOCALE_PREFIX)pt_PT/LC_MESSAGES/tuxpaint.mo
+	@install -d $(LOCALE_PREFIX)pt/LC_MESSAGES
+	@cp trans/pt_pt.mo $(LOCALE_PREFIX)pt/LC_MESSAGES/tuxpaint.mo
+	@chmod 644 $(LOCALE_PREFIX)pt/LC_MESSAGES/tuxpaint.mo
 	@#
 	@echo "   ro_RO ...Romanian..."
 	@install -d $(LOCALE_PREFIX)ro/LC_MESSAGES
--- tuxpaint-0.9.14/src/tuxpaint.desktop~	Wed Nov 10 12:57:16 2004
+++ tuxpaint-0.9.14/src/tuxpaint.desktop	Wed Nov 10 12:57:16 2004
@@ -5,7 +5,7 @@
 Name[be]=Малюй разам з Tux!
 Name[ca]=Tux Paint
 Name[da]=Tux maling
-Name[en_gb]=Tux Paint
+Name[en_GB]=Tux Paint
 Name[es]=Tux Paint
 Name[eu]=Tux Paint
 Name[fi]=Tux Paint
@@ -18,15 +18,15 @@
 Name[nl]=TuxPaint
 Name[nn]=Tux Paint
 Name[pl]=Tux Paint
-Name[pt_br]=Tux Paint
-Name[pt_pt]=Tux Paint
+Name[pt_BR]=Tux Paint
+Name[pt]=Tux Paint
 Name[ru]=Рисуй вместе с Tux!
 Name[sl]=Tux Paint
 Name[ta]=¼ìŠ ¦ÀÂ¢ýð
 Name[tr]=Tux Boyama
 Name[wa]=Tux Paint
-Name[zh_cn]=Tux Paint
-Name[zh_tw]=繪畫
+Name[zh_CN]=Tux Paint
+Name[zh_TW]=繪畫
 Type=Application
 Exec=tuxpaint
 Icon=tuxpaint
@@ -39,7 +39,7 @@
 GenericName[cy]=Rhaglen lunio
 GenericName[da]=Tegneprogram
 GenericName[de]=Malprogramm
-GenericName[en_gb]=Drawing program
+GenericName[en_GB]=Drawing program
 GenericName[es]=Programa de dibujo
 GenericName[eu]=Marrazketa programa
 GenericName[fi]=Piirto-ohjelma
@@ -55,16 +55,16 @@
 GenericName[nl]=Tekenprogramma
 GenericName[nn]=Teikneprogram
 GenericName[pl]=Program do rysowania
-GenericName[pt_br]=Programa de desenho
-GenericName[pt_pt]=Programa de desenho
+GenericName[pt_BR]=Programa de desenho
+GenericName[pt]=Programa de desenho
 GenericName[ru]=Программа для рисования
 GenericName[sl]=Program za slikanje
 GenericName[sv]=Ritprogram
 GenericName[ta]=Å¨ÃÔõ ¦ºÂÄ¢
 GenericName[tr]=Çizim programı
 GenericName[wa]=Programe di dessinaedje
-GenericName[zh_cn]=绘图程序
-GenericName[zh_tw]=畫圖程式
+GenericName[zh_CN]=绘图程序
+GenericName[zh_TW]=畫圖程式
 Comment=A drawing program for children.
 Comment[af]='n Tekenprogram vir kinders.
 Comment[be]=Дзіцячая праграма для маляваньня.
@@ -73,7 +73,7 @@
 Comment[cy]=Rhaglen lunio ar gyfer plant.
 Comment[da]=Et tegneprogram for børn
 Comment[de]=Ein Malprogramm für (kleine) Kinder.
-Comment[en_gb]=A drawing program for children.
+Comment[en_GB]=A drawing program for children.
 Comment[es]=Un programa de dibujo para niños.
 Comment[eu]=Umeentzako marrazketa programa
 Comment[fi]=Piirto-ohjelma pienille lapsille.
@@ -89,15 +89,15 @@
 Comment[nl]=Een tekenprogramma voor kinderen.
 Comment[nn]=Eit teikneprogram for dei yngste.
 Comment[pl]=Tux Paint: prosty program do rysowania dla dzieci.
-Comment[pt_br]=Um programa de desenho para crianças.
-Comment[pt_pt]=Um programa de desenho para crianças.
+Comment[pt_BR]=Um programa de desenho para crianças.
+Comment[pt]=Um programa de desenho para crianças.
 Comment[ru]=Детская программа для рисования.
 Comment[sl]=Otroški program za slikanje.
 Comment[sv]=Ett ritprogram för yngre barn.
 Comment[ta]=ÌÆó¨¾¸Ùì¸¡É ´Õ Å¨ÃÔõ ¦ºÂÄ¢
 Comment[tr]=Çocuklar için bir çizim programı
 Comment[wa]=Programe di dessinaedje po ls efants
-Comment[zh_cn]=孩子的绘图程序。
-Comment[zh_tw]=一個兒童專用的畫圖程式。
+Comment[zh_CN]=孩子的绘图程序。
+Comment[zh_TW]=一個兒童專用的畫圖程式。
 Categories=Education;Art;
 # vi: set encoding=utf-8
