--- Makefile.org	Sat Feb  1 00:54:24 2003
+++ Makefile	Wed Feb 12 09:41:20 2003
@@ -11,7 +11,7 @@
 
 # Where to install things:
 
-PREFIX=/usr/local
+PREFIX=$(_prefix)
 
 
 # Program:
@@ -51,7 +51,7 @@
 X11_ICON_PREFIX=$(PREFIX)/X11R6/include/X11/pixmaps/
 GNOME_PREFIX=`gnome-config --prefix`
 KDE_PREFIX=`kde-config --install apps --expandvars`
-KDE_ICON_PREFIX=`kde-config --install icon --expandvars`
+#KDE_ICON_PREFIX=`kde-config --install icon --expandvars`
 
 
 # Locale files
