--- tuxpaint-2002.09.29/Makefile.orig	Fri Oct 11 09:37:07 2002
+++ tuxpaint-2002.09.29/Makefile	Fri Oct 11 09:38:11 2002
@@ -68,7 +68,7 @@
 
 # The entire set of CFLAGS:
 
-CFLAGS=-O2 -Wall $(SDL_CFLAGS) -DDATA_PREFIX=\"$(DATA_PREFIX)\" \
+CFLAGS=$(OPTFLAGS) -Wall $(SDL_CFLAGS) -DDATA_PREFIX=\"$(DATA_PREFIX)\" \
 	-D$(NOSOUNDFLAG) -DDOC_PREFIX=\"$(DOC_PREFIX)\" \
 	-DLOCALEDIR=\"$(LOCALE_PREFIX)\" -DCONFDIR=\"$(CONFDIR)\"
 
