--- tuxpaint-0.9.21/src/tuxpaint.desktop~	2009-06-19 23:01:51.000000000 +0200
+++ tuxpaint-0.9.21/src/tuxpaint.desktop	2009-06-29 13:17:46.000000000 +0200
@@ -48,7 +48,7 @@
 Name[oc]=Tux Paint
 Name[pl]=Tux Paint
 Name[pt_BR]=Tux Paint
-Name[pt_PT]=Tux Paint
+Name[pt]=Tux Paint
 Name[ru]=Рисуй вместе с Tux!
 Name[sk]=Kreslenie s Tuxom
 Name[sl]=Tux Paint
@@ -125,7 +125,7 @@
 GenericName[oc]=Logicial de dessenh
 GenericName[pl]=Program do rysowania
 GenericName[pt_BR]=Programa de desenho
-GenericName[pt_PT]=Programa de desenho
+GenericName[pt]=Programa de desenho
 GenericName[ro]=Program de desenat
 GenericName[ru]=Программа для рисования
 GenericName[sk]=Program na kreslenie
@@ -197,7 +197,7 @@
 Comment[nr]=Iphrogremu yokudweba yabantwana.
 Comment[pl]=Program do rysowania dla dzieci.
 Comment[pt_BR]=Um programa de desenho para crianças.
-Comment[pt_PT]=Um programa de desenho para crianças.
+Comment[pt]=Um programa de desenho para crianças.
 Comment[ro]=Un program de desenat pentru copii
 Comment[ru]=Детская программа для рисования.
 Comment[sk]=Program na kreslenie pre deti.
